SECP-service/src/main/resources/assets/app/js/app.js

```diff
@@ -45,43 +56,64 @@ angular.module('SECP', ['ngResource', 'ngRoute', 'ui.bootstrap', 'ui.date', 'rou
         templateUrl: 'views/portal/portal.html',
         controller: 'PortalController',
         css: 'css/portal.css',
-        requiresLogin: true
+        requiresLogin: true,
+        resolve: {
+            isAdmin: isAdmin
+        }
       })
       .when('/portal/audit', {
         templateUrl: 'views/portal/audit.html',
         controller: 'PortalController',
         css: 'css/portal.css',
-        requiresLogin: true
+        requiresLogin: true,
+        resolve: {
+            isAdmin: isAdmin
+        }
       })
       .when('/portal/audit/user', {
           templateUrl: 'views/portal/audit-user.html',
           controller:'PortalController',
           css: 'css/portal.css',
-          requiresLogin: true
+          requiresLogin: true,
+          resolve: {
+            isAdmin: isAdmin
+          }
       })
       .when('/portal/audit/group', {
           templateUrl: 'views/portal/audit-group.html',
           controller:'PortalController',
           css: 'css/portal.css',
-          requiresLogin: true
+          requiresLogin: true,
+          resolve: {
+            isAdmin: isAdmin
+          }
       })
       .when('/portal/manage', {
           templateUrl: 'views/portal/manage.html',
           controller:'PortalController',
           css: 'css/portal.css',
-          requiresLogin: true
+          requiresLogin: true,
+          resolve: {
+            isAdmin: isAdmin
+          }
       })
       .when('/portal/manage/user', {
           templateUrl: 'views/portal/manage-user.html',
           controller:'PortalController',
           css: 'css/portal.css',
-          requiresLogin: true
+          requiresLogin: true,
+          resolve: {
+            isAdmin: isAdmin
+          }
       })
       .when('/portal/manage/group', {
           templateUrl: 'views/portal/manage-group.html',
           controller:'PortalController',
           css: 'css/portal.css',
-          requiresLogin: true
+          requiresLogin: true,
+          resolve: {
```

Comment:
Nice, didn't know we can set the values here like that.

Reply:
yeah we can. The route wont be loaded unless that promise evaluates to true.